purchase_order_type/models/purchase_order.py

```diff
@@ -21,7 +21,6 @@ def _default_order_type(self):
         states=Purchase.READONLY_STATES,
         string="Type",
         ondelete="restrict",
-        domain="[('company_id', 'in', [False, company_id])]",
```

Comment:
Why do you remove this? Please explain, for example in the commit message body... Also, please provide a clearer commit message (title)

Reply:
This shouldn't be removed. No matter record rules, you can have more than one company displayed on your user, and thus, this will show incorrect types.

Reply:
I still see this missing

Reply:
Sorry, it is already restored.